src/EventHub/EventHub.Autorest/custom/New-AzEventHubAuthorizationRule.ps1

```diff
@@ -62,7 +62,7 @@ function New-AzEventHubAuthorizationRule{
 
         [Parameter(HelpMessage = "The rights associated with the rule.")]
         [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Body')]
-        [System.String]
+        [System.String[]]
```

Comment:
Was this a bug in the already existing code ? Shouldn't this have been an array before also ?